### PR TITLE
Displays lobby theme name when playing.

### PR DESCRIPTION
**Pull request by ghost**
:cl: McBawbaggings
tweak: The game now displays the current lobby theme when playing.
/:cl:
image

Idea suggested by JohnGinnane: https://hippiestation.com/threads/post-lobby-and-round-end-music-when-it-plays.8575/

### CODE REVIEW

**Comment by ghost** [comment]
You currently have a negative Fix/Feature pull request delta of -2. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by earthmoonjune** [comment]
+1

**Comment by ghost** [comment]
got an idea to fully modularise will do tomorrow morning

**Comment by ghost** [comment]
+1

**Comment by ExpAntique** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by JohnGinnane** [comment]
hey i wanted to do this :(

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -20,6 +20,7 @@ SUBSYSTEM_DEF(ticker)
 	var/event = 0
 
 	var/login_music							//music played in pregame lobby
+	var/login_music_name					// hippie - song name displayed when title theme plays
 	var/round_end_sound						//music/jingle played when the world reboots
```
Comment (Carbonhell) [inline code/controllers/subsystem/ticker.dm]: Modularize?

**Review by Carbonhell — CHANGES_REQUESTED** [review]

**Comment by Carbonhell** [comment]
Only issue is that some oggs have nondescriptive names, but it's all good

**Comment by JohnGinnane** [comment]
yeah to get around that I was going to add in "You can find this song here: https://github.com/HippieStation/HippieStation/tree/master/hippiestation/sound/lobby"

**Comment by JohnGinnane** [comment]
image
Not immediately sure if there's a way around the space issue and I noticed a fullstop at the end for some reason

**Comment by ghost** [comment]
tried wrapping it inside an href but i couldn't get it to work nicely
image

**Comment by JohnGinnane** [comment]
what's wong with what you have there?

**Comment by ghost** [comment]
it's like a personal computer link and doesn't work properly idk how to explain it

**Comment by Pyko1** [comment]
sure +1

**Review by Carbonhell — DISMISSED** [review]

**Review by praisenarsie — APPROVED** [review]

**Comment by GuyonBroadway** [comment]
Oh this has a +2 and successful code review, how did THAT slip by?

**Pull request merged**